src/Amalgam/interpreter/InterpreterOpcodesMath.cpp

```diff
@@ -805,6 +806,32 @@ EvaluableNodeReference Interpreter::InterpretNode_ENT_MIN(EvaluableNode *en, boo
 	return EvaluableNodeReference::Null();
 }
 
+EvaluableNodeReference Interpreter::InterpretNode_ENT_INDEX_MAX(EvaluableNode *en, bool immediate_result)
+{
+	auto &ocn = en->GetOrderedChildNodes();
+
+	if(ocn.size() == 0)
+		return EvaluableNodeReference::Null();
+
+	if(ocn[0] != nullptr && ocn[0]->GetType() == ENT_ASSOC)
+		return InterpretNode_IndexMinMaxAssoc(ocn[0], std::greater(), -std::numeric_limits<double>::infinity(), immediate_result);
+	else
+		return InterpretNode_IndexMinMaxList(en, std::greater(), -std::numeric_limits<double>::infinity(), immediate_result);
```

Comment:
I don't think I can easily make this static because  using std::greater/less obligates me to make it a template. Unless there is some way I am not thinking of?